pkg/utils/utils.go

```diff
@@ -397,3 +398,15 @@ func isFileSystemCaseInsensitive() bool {
 	// On Linux, file systems are typically case sensitive
 	return filepath.Separator == '\\' || strings.Contains(strings.ToLower(os.Getenv("OS")), "darwin")
 }
+
+// This exclusive function returns the standard output and standard error as strings
+func ExecuteCommand(program string, args ...string) (string, string, error) {
```

Comment:
Wouldn't be better to place this function next to the "normal" `ExecuteCommand` in [exec.go](https://github.com/Open-CMSIS-Pack/cbuild/blob/main/pkg/utils/exec.go#L32)?